.github/workflows/github-actions-cron-update-OR.yml

```diff
@@ -0,0 +1,40 @@
+name: Create draft PR for updated OpenROAD submodule
+on:
+  schedule:
+  - cron: "0 8 * * *"
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out repository code recursively
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Pull changes in OpenROAD submodule
+        id: remote-update
+        run: |
+          cd tools/OpenROAD
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          echo "::set-output name=has_update::$(git --no-pager diff --name-only origin/master..HEAD)"
+          git checkout master
+          git pull
```

Comment:
Are we sure on the order here? Should we do a `fetch` before setting the `has_update` value?